release_notes/43-release-notes.adoc

```diff
@@ -327,6 +328,26 @@ It also provides the following security fixes:
 * Helm: Missing YAML content leads to panic link:https://access.redhat.com/security/cve/cve-2024-26147[(CVE-2024-26147)]
 * Helm: Shows secrets with `--dry-run` option in clear text link:https://access.redhat.com/security/cve/cve-2019-25210[(CVE-2019-25210)]
 
+[id="resolved-in-version-437_{context}"]
+=== Resolved in version 4.3.7
+
+*Release date*: 8 May 2024
+
+This release provides the following bug fixes:
+
+* This release fixes an issue where the Central pod failed with a `SQLSTATE 23503` error after upgrading {product-title-short} to release 4.3.6. This resulted in the collector pods showing the `CrashLoopBackoff` state because the collector probe was unavailable.
+* This release updates the Scanner baseline vulnerability data to address changes made to the Red Hat security data feeds that were not compatible with earlier data from Scanner's scheduled feed processing. This fixes various issues where vulnerabilities were detected for images containing packages that were previously incorrectly indicated as affected by a vulnerability.
```

Comment:
Suggestion - I think the double adverb here sounds a little awkward, imo the past tense might be enough to indicate that this was previous behavior:
```suggestion
* This release updates the Scanner baseline vulnerability data to address changes made to the Red Hat security data feeds that were not compatible with earlier data from Scanner's scheduled feed processing. This fixes various issues where vulnerabilities were detected for images containing packages that were incorrectly indicated as affected by a vulnerability.
```